content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -268,32 +278,36 @@ once bound to a node, a Pod will never be rebound to another node.
 
 ## Pod lifetime
 
-In general, Pods remain until a human or controller process explicitly removes them.
-The control plane cleans up terminated Pods (with a phase of `Succeeded` or
+In general, Pods remain until a human or
+{{< glossary_tooltip term_id="controller" text="controller" >}} process
+explicitly removes them.
+The control plane cleans up terminated Pods (with a phase of `Succeeded` or 
 `Failed`), when the number of Pods exceeds the configured threshold
 (determined by `terminated-pod-gc-threshold` in the kube-controller-manager).
 This avoids a resource leak as Pods are created and terminated over time.
 
-Three types of controllers are available:
+There are different kinds of resources for creating Pods:
 
-- Use a [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/) for Pods that are expected to terminate,
+- Use a {{< glossary_tooltip term_id="deployment" >}},
+  {{< glossary_tooltip term_id="replica-set" >}} or {{< glossary_tooltip term_id="statefulset" >}}
+  for Pods that are not expected to terminate, for example, web servers.
+
+- Use a {{< glossary_tooltip term_id="job" >}}
+  for Pods that are expected to terminate once their work is complete;
   for example, batch computations. Jobs are appropriate only for Pods with
   `restartPolicy` equal to OnFailure or Never.
 
-- Use a [ReplicationController](/docs/concepts/workloads/controllers/replicationcontroller/),
-  [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/), or
-  [Deployment](/docs/concepts/workloads/controllers/deployment/)
-  for Pods that are not expected to terminate, for example, web servers.
-  ReplicationControllers are appropriate only for Pods with a `restartPolicy` of
-  Always.
+- Use a {{< glossary_tooltip term_id="daemonset" >}}
+  for Pods that need to run one per eligible node, because they provide a
+  machine-specific system service.
 
-- Use a [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) for Pods that need to run one per
-  machine, because they provide a machine-specific system service.
+On their own, Pods are not resilient to machine failures. Most workload
+resources, such as ReplicaSet, have a control loop that is responsible for
+recovering from a Node- or Pod-level failure.
```

Comment:
Suggest removing this paragraph or link it to the place in k8s docs where continuous availability is discussed for pods wrt or in spite of machine failures. Iit introduces an entirely new very large topic, machine failures, without explaining the types of machine failures or what is meant by machine failures. A machine failure does not change the pod's lifecycle but it may trigger its abrupt end. Pods will be resilient to some/many types of machine failures. Does not touch on container failures or pod specific resources.

I know what you mean here but it introduces more questions than it addresses. Is there a place in the docs where node/pod/container failover is discussed talking about pod resiliency? If so could have a Note: comment that links to it?

Reply:
I've logged https://github.com/kubernetes/website/issues/19404 to track that page (about fault recover), now that I've realized there isn't one.

Reply:
resolved